samples/peripheral/radio_test/sample.yaml

```diff
@@ -35,18 +35,34 @@ tests:
   sample.peripheral.radio_test.nrf5340_nrf21540_usb:
     sysbuild: true
     build_only: true
-    extra_args: SHIELD=nrf21540ek FILE_SUFFIX=usb
+    extra_args: radio_test_SHIELD=nrf21540ek remote_shell_SHIELD=nrf21540ek_fwd FILE_SUFFIX=usb
     integration_platforms:
       - nrf5340dk/nrf5340/cpunet
     platform_allow: nrf5340dk/nrf5340/cpunet
     tags: ci_build sysbuild
   sample.peripheral.radio_test.nrf5340_nrf21540.no_automatic_power:
     sysbuild: true
     build_only: true
-    extra_args: SHIELD=nrf21540ek
+    extra_args: radio_test_SHIELD=nrf21540ek remote_shell_SHIELD=nrf21540ek_fwd
     extra_configs:
       - CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC=n
     integration_platforms:
       - nrf5340dk/nrf5340/cpunet
     platform_allow: nrf5340dk/nrf5340/cpunet
     tags: ci_build sysbuild
+  sample.peripheral.radio_test.nrf5340_nrf2220:
+    sysbuild: true
+    build_only: true
+    extra_args: radio_test_SHIELD=nrf2220ek remote_shell_SHIELD=nrf2220ek_fwd
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpunet
+    platform_allow: nrf5340dk/nrf5340/cpunet
+    tags: ci_build sysbuild
+  sample.peripheral.radio_test.nrf5340_nrf2240:
+    sysbuild: true
+    build_only: true
+    extra_args: radio_test_SHIELD=nrf2240ek remote_shell_SHIELD=nrf2240ek_fwd
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpunet
+    platform_allow: nrf5340dk/nrf5340/cpunet
+    tags: ci_build sysbuild
```

Comment:
This PR is a bugfix. The release is in the bugfix state. Adding more features like supported shields to samples is not good.